src/Security/SecurityConfigUpdater.php

```diff
@@ -30,6 +30,27 @@ public function __construct(
     ) {
     }
 
+    public function updateForFormLogin(string $yamlSource, string $firewallToUpdate, string $loginPath, string $checkPath): string
+    {
+        $this->manipulator = new YamlSourceManipulator($yamlSource);
+
+        if (null !== $this->ysmLogger) {
+            $this->manipulator->setLogger($this->ysmLogger);
+        }
```

Comment:
Minor, but I guess we should add a private `createYamlManipulator()` in this class that would instantiate and set the logger all at once.